astroquery/astrometry_net/core.py

```diff
@@ -412,6 +415,11 @@ def solve_from_image(self, image_file_path, *, force_image_upload=False,
                                          cache=False,
                                          files={'file': f})
         else:
+            warning_msg = "Removing photutils functionality to obtain extracted positions list from " \
+                          "AstrometryNetClass.solve_from_source_list. Users will need to " \
+                          "submit pre-extracted catalog positions or a fits file for https://nova.astrometry.net/ " \
+                          "to extract with their algorithm."
```

Comment:
Nitpick, but I would rephrase this a bit, as it's the user who sees this message not a third party.
```suggestion
            warning_msg = ("Running photutils functionality within astroquery to obtain extracted positions list from " 
                          "AstrometryNetClass.solve_from_source_list() is deprecated and will be removed in the next release. "
                          "You need to submit catalog positions or a fits file to obtain an astrometric solution using this service.")
```